### PR TITLE
Responsive 1

### PR DESCRIPTION
**Pull request by alicemcneil**
made my webpage responsive until width:460px

### CODE REVIEW

File: css/normalize.css [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+textarea {
+  overflow: auto;
+}
+
+/**
+ * 1. Add the correct box sizing in IE 10.
+ * 2. Remove the padding in IE 10.
+ */
+
+[type="checkbox"],
```
Comment (sjr-mayden) [inline css/normalize.css:270]: Adding an input here would make it more specific

File: css/normalize.css [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
+[type="radio"] {
```
Comment (sjr-mayden) [inline css/normalize.css:271]: Adding an input here would make it more specific

File: css/normalize.css [hunk 3]
```diff
@@ -0,0 +1,349 @@
...
+  box-sizing: border-box; /* 1 */
+  padding: 0; /* 2 */
+}
+
+/**
+ * Correct the cursor style of increment and decrement buttons in Chrome.
+ */
+
+[type="number"]::-webkit-inner-spin-button,
```
Comment (sjr-mayden) [inline css/normalize.css:280]: Adding an input here would make it more specific

File: css/normalize.css [hunk 4]
```diff
@@ -0,0 +1,349 @@
...
+[type="number"]::-webkit-outer-spin-button {
```
Comment (sjr-mayden) [inline css/normalize.css:281]: Adding an input here would make it more specific

File: css/normalize.css [hunk 5]
```diff
@@ -0,0 +1,349 @@
...
+  height: auto;
+}
+
+/**
+ * 1. Correct the odd appearance in Chrome and Safari.
+ * 2. Correct the outline style in Safari.
+ */
+
+[type="search"] {
```
Comment (sjr-mayden) [inline css/normalize.css:290]: Adding an input here would make it more specific

File: css/normalize.css [hunk 6]
```diff
@@ -0,0 +1,349 @@
...
+  -webkit-appearance: textfield; /* 1 */
+  outline-offset: -2px; /* 2 */
+}
+
+/**
+ * Remove the inner padding in Chrome and Safari on macOS.
+ */
+
+[type="search"]::-webkit-search-decoration {
```
Comment (sjr-mayden) [inline css/normalize.css:299]: Adding an input here would make it more specific

File: css/normalize.css [hunk 7]
```diff
@@ -0,0 +1,349 @@
...
+
+template {
+  display: none;
+}
+
+/**
+ * Add the correct display in IE 10.
+ */
+
+[hidden] {
```
Comment (sjr-mayden) [inline css/normalize.css:347]: Adding/listing elements here would make this more specific

File: css/portfolio.css [hunk 8]
```diff
@@ -0,0 +1,641 @@
...
+* {
+    box-sizing: border-box
+}
+
+body {
+    font-family: 'Heebo Black', sans-serif;
+    text-transform: uppercase;
+    letter-spacing: 2px;
+    background-image: url("../images/teal.png");
+    /*width:100vw;*/
```
Comment (sjr-mayden) [inline css/portfolio.css]: Delete commented code

File: css/portfolio.css [hunk 9]
```diff
@@ -0,0 +1,641 @@
...
+
```
Comment (sjr-mayden) [inline css/portfolio.css]: Unnecessary extra line

File: css/portfolio.css [hunk 10]
```diff
@@ -0,0 +1,641 @@
...
+    color: white;
+    position: relative;
+    font-size: 62%;
+
```
Comment (sjr-mayden) [inline css/portfolio.css]: Unnecessary extra line

File: css/portfolio.css [hunk 11]
```diff
@@ -0,0 +1,641 @@
...
+}
+
+.container {
+    margin-right: auto; /* 1 */
+    margin-left:  auto; /* 1 */
+
```
Comment (sjr-mayden) [inline css/portfolio.css]: Unnecessary extra line

File: css/portfolio.css [hunk 12]
```diff
@@ -0,0 +1,641 @@
...
+    max-width: 1200px; /* 2 */
+
```
Comment (sjr-mayden) [inline css/portfolio.css]: Unnecessary extra line

File: css/portfolio.css [hunk 13]
```diff
@@ -0,0 +1,641 @@
...
+    padding-right: 10px; /* 3 */
+    padding-left:  10px; /* 3 */
+
```
Comment (sjr-mayden) [inline css/portfolio.css]: Unnecessary extra line

File: css/portfolio.css [hunk 14]
```diff
@@ -0,0 +1,641 @@
...
+}
+
+a {
+    text-decoration: none;
+    color: white;
+}
+
```
Comment (sjr-mayden) [inline css/portfolio.css:30]: Unnecessary extra line

File: css/portfolio.css [hunk 15]
```diff
@@ -0,0 +1,641 @@
...
+
+a:hover {
+    color: #2b7a78;
+    transition: 1s;
+}
+
```
Comment (sjr-mayden) [inline css/portfolio.css:36]: Unnecessary extra line

File: css/portfolio.css [hunk 16]
```diff
@@ -0,0 +1,641 @@
...
+
```
Comment (sjr-mayden) [inline css/portfolio.css:37]: Unnecessary extra line

File: css/portfolio.css [hunk 17]
```diff
@@ -0,0 +1,641 @@
...
+
```
Comment (sjr-mayden) [inline css/portfolio.css]: Unnecessary extra line

File: css/portfolio.css [hunk 18]
```diff
@@ -0,0 +1,641 @@
...
+    z-index:6;
+}
+
+.logo {
+    position: absolute;
+    z-index: 10;
+    top: 30px;
+    left: 40px;
+    width: 250px;
+
```
Comment (sjr-mayden) [inline css/portfolio.css]: Unnecessary extra line

File: css/portfolio.css [hunk 19]
```diff
@@ -0,0 +1,641 @@
...
+}
+
```
Comment (sjr-mayden) [inline css/portfolio.css:80]: Unnecessary extra line

File: css/portfolio.css [hunk 20]
```diff
@@ -0,0 +1,641 @@
...
+
```
Comment (sjr-mayden) [inline css/portfolio.css:81]: Unnecessary extra line

File: css/portfolio.css [hunk 21]
```diff
@@ -0,0 +1,641 @@
...
+    padding-right: 25px;
+    font-size: 1rem;
+}
+
+/*   MAIN HEAD  */
+
+.overflow {
+    overflow: auto;
+}
+
```
Comment (sjr-mayden) [inline css/portfolio.css:108]: Unnecessary extra line

File: css/portfolio.css [hunk 22]
```diff
@@ -0,0 +1,641 @@
...
+
```
Comment (sjr-mayden) [inline css/portfolio.css]: Unnecessary extra line

File: css/portfolio.css [hunk 23]
```diff
@@ -0,0 +1,641 @@
...
+
+h2 {
+    color: #2b7a78;
+    letter-spacing: 10px;
+    margin-top: 300px;
+    text-align: right;
+    font-size: 2rem;
+
```
Comment (sjr-mayden) [inline css/portfolio.css]: Unnecessary extra line

File: css/portfolio.css [hunk 24]
```diff
@@ -0,0 +1,641 @@
...
+}
+
```
Comment (sjr-mayden) [inline css/portfolio.css:116]: Unnecessary extra line

File: css/portfolio.css [hunk 25]
```diff
@@ -0,0 +1,641 @@
...
+
+h1 {
+    font-size:10rem;
+    letter-spacing: 40px;
+    z-index: 20;
+    text-align: center;
+}
+
```
Comment (sjr-mayden) [inline css/portfolio.css:123]: Unnecessary extra line

File: css/portfolio.css [hunk 26]
```diff
@@ -0,0 +1,641 @@
...
+
```
Comment (sjr-mayden) [inline css/portfolio.css:124]: Unnecessary extra line

File: css/portfolio.css [hunk 27]
```diff
@@ -0,0 +1,641 @@
...
+
+/*    TOP SECTION    */
+
+/*.top-section {*/
+    /*padding: 20px 60px;*/
+    /*margin-left: 10%;*/
+
```
Comment (sjr-mayden) [inline css/portfolio.css]: Unnecessary extra line

File: css/portfolio.css [hunk 28]
```diff
@@ -0,0 +1,641 @@
...
+    background-color: #def2f1;
+    width: 200px;
+    height: 200px;
+    float: left;
+    /*margin-top: 40px;*/
+    /*margin-left: 200px;*/
+    margin-right: 130px;
+    /*margin-left: 180px;*/
+    /*margin-bottom: 100px;*/
+
```
Comment (sjr-mayden) [inline css/portfolio.css]: Unnecessary extra line

File: css/portfolio.css [hunk 29]
```diff
@@ -0,0 +1,641 @@
...
+}
+
+h3 + p {
+    text-transform: none;
+    text-align: left;
+    line-height: 30px;
+    font-family: sans-serif;
+    font-weight: bold;
+    font-size: 1rem;
+    /*padding: 30px;*/
```
Comment (sjr-mayden) [inline css/portfolio.css]: Delete commented code

File: css/portfolio.css [hunk 30]
```diff
@@ -0,0 +1,641 @@
...
+
```
Comment (sjr-mayden) [inline css/portfolio.css]: Unnecessary extra line

File: css/portfolio.css [hunk 31]
```diff
@@ -0,0 +1,641 @@
...
+.portfolio-title {
+    background-color: #2b7a78;
+    display: inline-block;
+    padding: 30px 65px;
+    letter-spacing: 5px;
+    font-size: 1.5rem;
+    margin-bottom: 0;
+}
+
```
Comment (sjr-mayden) [inline css/portfolio.css:182]: Unnecessary extra line

File: css/portfolio.css [hunk 32]
```diff
@@ -0,0 +1,641 @@
...
+
+/*    BACKGROUND BOXES     */
+
```
Comment (sjr-mayden) [inline css/portfolio.css:184]: Unnecessary extra line

File: css/portfolio.css [hunk 33]
```diff
@@ -0,0 +1,641 @@
...
+
+.project-link {
```
Comment (sjr-mayden) [inline css/portfolio.css]: Something is wrong with the indentation of this class

File: css/portfolio.css [hunk 34]
```diff
@@ -0,0 +1,641 @@
...
+    position: absolute;
+   margin-top: 150px;
+}
+
+/*.color-block {*/
```
Comment (sjr-mayden) [inline css/portfolio.css]: Delete commented code

File: css/portfolio.css [hunk 35]
```diff
@@ -0,0 +1,641 @@
...
+/*}*/
+
+
+.image-container {
+    height: 700px;
+    z-index: 10;
+    filter: grayscale(100%); /*check this for browser support (webkit?)*/
+    text-align: center;r
+}
+
```
Comment (sjr-mayden) [inline css/portfolio.css:192]: Unnecessary extra line

File: css/portfolio.css [hunk 36]
```diff
@@ -0,0 +1,641 @@
...
+.image2  {
+    height: 50%;
+    width: 50%;
+    float: left;
+    padding: 10px;
+}
+
+
```
Comment (sjr-mayden) [inline css/portfolio.css]: Unnecessary extra line

File: css/portfolio.css [hunk 37]
```diff
@@ -0,0 +1,641 @@
...
+}
+
+.image2:hover {
+    /*-webkit-transform: scale(1.3);*/
+    /*-ms-transform: scale(1.3);*/
+    transform: scale(1.3);
+    transition: 1s;
+}
+
+
```
Comment (sjr-mayden) [inline css/portfolio.css:221]: Unnecessary extra line

File: css/portfolio.css [hunk 38]
```diff
@@ -0,0 +1,641 @@
...
+    transition: 1s;
+}
+
+.image2:hover {
+    /*-webkit-transform: scale(1.3);*/
+    /*-ms-transform: scale(1.3);*/
+    transform: scale(1.3);
+    transition: 1s;
+}
+
```
Comment (sjr-mayden) [inline css/portfolio.css:220]: Unnecessary extra line

File: css/portfolio.css [hunk 39]
```diff
@@ -0,0 +1,641 @@
...
+
+
```
Comment (sjr-mayden) [inline css/portfolio.css]: Unnecessary extra line

File: css/portfolio.css [hunk 40]
```diff
@@ -0,0 +1,641 @@
...
+
```
Comment (sjr-mayden) [inline css/portfolio.css]: Unnecessary extra line

File: css/portfolio.css [hunk 41]
```diff
@@ -0,0 +1,641 @@
...
+
```
Comment (sjr-mayden) [inline css/portfolio.css]: Unnecessary extra line

File: css/portfolio.css [hunk 42]
```diff
@@ -0,0 +1,641 @@
...
+
```
Comment (sjr-mayden) [inline css/portfolio.css]: Unnecessary extra line

File: css/portfolio.css [hunk 43]
```diff
@@ -0,0 +1,641 @@
...
+/*  CONTACT SECTION  */
+
+.contact-link {
+    position: absolute;
+    margin-top: 1200px;
+
```
Comment (sjr-mayden) [inline css/portfolio.css:225]: Unnecessary extra line

File: css/portfolio.css [hunk 44]
```diff
@@ -0,0 +1,641 @@
...
+}
+
```
Comment (sjr-mayden) [inline css/portfolio.css:227]: Unnecessary extra line

File: css/portfolio.css [hunk 45]
```diff
@@ -0,0 +1,641 @@
...
+.icon:hover {
+    background-color: #2b7a78 ;
+    transition: 1s;
+    padding: 10px;
+    border-radius: 8px;
+}
+
+.git {
```
Comment (sjr-mayden) [inline css/portfolio.css]: github, not git

File: css/portfolio.css [hunk 46]
```diff
@@ -0,0 +1,641 @@
...
+    margin-left: 0;
+}
+
```
Comment (sjr-mayden) [inline css/portfolio.css:289]: Unnecessary extra line

File: css/portfolio.css [hunk 47]
```diff
@@ -0,0 +1,641 @@
...
+
```
Comment (sjr-mayden) [inline css/portfolio.css]: Unnecessary extra line

File: css/portfolio.css [hunk 48]
```diff
@@ -0,0 +1,641 @@
...
+    height: 0;
+    border-left: 900px solid transparent;
+    border-bottom: 500px solid #2b7a78;
+    position: absolute;
+    bottom: 0;
+    right: 0;
+    z-index: 5;
+    top: 2600px;
+}
+
```
Comment (sjr-mayden) [inline css/portfolio.css]: Unnecessary extra line

File: css/portfolio.css [hunk 49]
```diff
@@ -0,0 +1,641 @@
...
+
```
Comment (sjr-mayden) [inline css/portfolio.css]: Unnecessary extra line

File: css/portfolio.css [hunk 50]
```diff
@@ -0,0 +1,641 @@
...
+
```
Comment (sjr-mayden) [inline css/portfolio.css]: Unnecessary extra line

File: css/portfolio.css [hunk 51]
```diff
@@ -0,0 +1,641 @@
...
+
```
Comment (sjr-mayden) [inline css/portfolio.css:299]: Unnecessary extra line

File: css/portfolio.css [hunk 52]
```diff
@@ -0,0 +1,641 @@
...
+        /*display: none;*/
+    /*}*/
+    .about-me p {
+        width: 93%;
+        font-size: 0.9rem;
+    }
+
+}
+
+
```
Comment (sjr-mayden) [inline css/portfolio.css:337]: Unnecessary extra line

File: css/portfolio.css [hunk 53]
```diff
@@ -0,0 +1,641 @@
...
+        width: 1000px;
+    }
+    .bottom-triangle {
+       top: 2500px;
+       border-left: 900px solid transparent;
+       border-bottom: 500px solid #2b7a78;
+    }
+    .contact-link {
+        margin-top: 250px;
+}
```
Comment (sjr-mayden) [inline css/portfolio.css]: Incorrect indentation

File: css/portfolio.css [hunk 54]
```diff
@@ -0,0 +1,641 @@
...
+}
+
```
Comment (sjr-mayden) [inline css/portfolio.css:373]: Unnecessary extra line

File: css/portfolio.css [hunk 55]
```diff
@@ -0,0 +1,641 @@
...
+
+@media only screen and (max-width: 990px) {
```
Comment (sjr-mayden) [inline css/portfolio.css:376]: Blank lines between all rules

File: css/portfolio.css [hunk 56]
```diff
@@ -0,0 +1,641 @@
...
+    .top-section {
+        margin-left: 10px;
+    }
+    .icon {
+        margin-left: 50px;
+    }
+    .image-container {
+        height: 500px;
+    }
+
```
Comment (sjr-mayden) [inline css/portfolio.css]: Unnecessary extra line

File: css/portfolio.css [hunk 57]
```diff
@@ -0,0 +1,641 @@
...
+}
+
+@media only screen and (max-width: 768px) {
```
Comment (sjr-mayden) [inline css/portfolio.css:410]: Blank lines between all rules

File: css/portfolio.css [hunk 58]
```diff
@@ -0,0 +1,641 @@
...
+    .project-link {
+        margin-top: 15px;
+    }
+    .bottom-triangle {
+        margin-top: 100px;
+    }
+    .contact-link {
+        margin-bottom: 100px;
+    }
+
```
Comment (sjr-mayden) [inline css/portfolio.css]: Unnecessary extra line

File: css/portfolio.css [hunk 59]
```diff
@@ -0,0 +1,641 @@
...
+}
+
+@media only screen and (max-width: 576px) {
```
Comment (sjr-mayden) [inline css/portfolio.css:469]: Blank lines between all rules

File: css/portfolio.css [hunk 60]
```diff
@@ -0,0 +1,641 @@
...
+        margin-left: 10px;
+        margin-bottom: 30px;
+    }
+    .bottom-triangle {
+        border: none;
+        width: 1000px;
+        height: 500px;
+        background-color: #2b7a78;
+    }
+
```
Comment (sjr-mayden) [inline css/portfolio.css:520]: Unnecessary extra line

File: css/portfolio.css [hunk 61]
```diff
@@ -0,0 +1,641 @@
...
+}
+
+@media only screen and (max-width: 450px) {
```
Comment (sjr-mayden) [inline css/portfolio.css:529]: Blank lines between all rules

File: html/portfolio1.html [hunk 62]
```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+
```
Comment (sjr-mayden) [inline html/portfolio1.html]: Unnecessary extra line

File: html/portfolio1.html [hunk 63]
```diff
@@ -0,0 +1,149 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+<!--LINKS-->
+    <link href="../css/normalize.css" rel="stylesheet" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Heebo" rel="stylesheet">
+    <link href="../css/portfolio.css" rel="stylesheet" type="text/css">
+
```
Comment (sjr-mayden) [inline html/portfolio1.html]: Unnecessary extra line

File: html/portfolio1.html [hunk 64]
```diff
@@ -0,0 +1,149 @@
...
+
+    <title>Alice McNeil</title>
+
+</head>
+
+<body>
+
+
```
Comment (sjr-mayden) [inline html/portfolio1.html]: Unnecessary extra line

File: html/portfolio1.html [hunk 65]
```diff
@@ -0,0 +1,149 @@
...
+
+    <!--LOGO-->
+
+        <img src="../icons/alice-logo.png" class="logo" alt="alice logo">
+
```
Comment (sjr-mayden) [inline html/portfolio1.html]: Unnecessary extra line

File: html/portfolio1.html [hunk 66]
```diff
@@ -0,0 +1,149 @@
...
+        <a class="triangle" name="home"></a>
+        <a class="square" name="home"></a>
+
+
```
Comment (sjr-mayden) [inline html/portfolio1.html]: Unnecessary extra line

File: html/portfolio1.html [hunk 67]
```diff
@@ -0,0 +1,149 @@
...
+    <div class="container">
+
+    <!--MAIN HEAD-->
+        <header>
+            <h2>Alice McNeil</h2>
+            <h1>Portfolio</h1>
+
+        </header>
+
```
Comment (sjr-mayden) [inline html/portfolio1.html]: Unnecessary extra line

File: html/portfolio1.html [hunk 68]
```diff
@@ -0,0 +1,149 @@
...
+                    I hope to learn the skills I will need to become a front-end software developer. I love graphic design, music and code!
+                    Please take a look around at my portfolio.</p>
+            </div>
+        </div>
+
+    <!--PORTFOLIO-->
+
+        <a name="projects" class="project-link"></a>
+
+
```
Comment (sjr-mayden) [inline html/portfolio1.html]: Unnecessary extra line

File: html/portfolio1.html [hunk 69]
```diff
@@ -0,0 +1,149 @@
...
+            <div class="my-work">
+
+                <h3 class="portfolio-title">Here's some of my projects</h3>
+            </div>
+
+         <!--IMAGES-->
+                <div class="image-container">
+
+
```
Comment (sjr-mayden) [inline html/portfolio1.html]: Unnecessary extra line

File: html/portfolio1.html [hunk 70]
```diff
@@ -0,0 +1,149 @@
...
+                        <img src="../images/merch.png" alt="rocketmakers merch" class="image1">
+
+
```
Comment (sjr-mayden) [inline html/portfolio1.html]: Unnecessary extra line

File: html/portfolio1.html [hunk 71]
```diff
@@ -0,0 +1,149 @@
...
+
+                    <img src="../images/pilotscreenshot.png" alt="pilotshop webpage" class="image2">
+                    <img src="../images/mayden.png" alt="mayden academy logo" class="image2">
+                </div>
+
+
+
```
Comment (sjr-mayden) [inline html/portfolio1.html]: Unnecessary extra line

File: html/portfolio1.html [hunk 72]
```diff
@@ -0,0 +1,149 @@
...
+            <div class="color-block">
+                <div class="dark-left"></div>
+                <div class="clear"></div>
+                <div class="light-right"></div>
+                <div class="white-low"></div>
+            </div>
+
+
+
```
Comment (sjr-mayden) [inline html/portfolio1.html]: Unnecessary extra line

File: html/portfolio1.html [hunk 73]
```diff
@@ -0,0 +1,149 @@
...
+
+        <!--CONTACT-->
+
+        <a name="contact" class="contact-link"></a>
+
+
```
Comment (sjr-mayden) [inline html/portfolio1.html]: Unnecessary extra line

File: html/portfolio1.html [hunk 74]
```diff
@@ -0,0 +1,149 @@
...
+        <div class="contact-section">
+            <h3>Contact me</h3>
+            <p class="get-in-touch">Well you've made it this far, you may as well get in touch with me.</p>
+            <a href="mailto:alicemcneil@live.co.uk" class="email">EMAIL: alicemcneil@live.co.uk</a>
+
```
Comment (sjr-mayden) [inline html/portfolio1.html]: Unnecessary extra line

File: html/portfolio1.html [hunk 75]
```diff
@@ -0,0 +1,149 @@
...
+
+        </div>
+
```
Comment (sjr-mayden) [inline html/portfolio1.html]: Unnecessary extra line

File: html/portfolio1.html [hunk 76]
```diff
@@ -0,0 +1,149 @@
...
+                <img src="../icons/dribble.svg" alt="dribble" class="icon">
+            </a>
+
+            <a href="https://www.linkedin.com/in/alice-mcneil-5805a5169/">
+                <img src="../icons/linkedin.svg" alt="linkedin" class="icon">
+            </a>
+
+        </div>
+
+
```
Comment (sjr-mayden) [inline html/portfolio1.html]: Unnecessary extra line

**Review by sjr-mayden — CHANGES_REQUESTED** [review]

File: css/portfolio.css (hunk 34 above)
Comment (alicemcneil) [reply css/portfolio.css]: this is still needed, it is my background shapes

**Pull request merged**